af/fastpath/fastpath/core.py

```diff
@@ -494,6 +494,26 @@ def score_measurement_facebook_messenger(msm):
     return scores
 
 
+def _extract_tcp_connect(tk):
+    # https://github.com/ooni/spec/blob/master/data-formats/df-005-tcpconnect.md
+    # NOTE: this is *NOT* ts-008-tcp-connect.md
```

Comment:
Yeah, I know, this is confusing :-(

Reply:
Since we can rename test helpers without too much pain, maybe `df-005-tcpdial.md`?

Reply:
We should eventually relocate all legacy tests to some special tree or mark them as deprecated in a more obvious way. I would not worry to much about the naming clash as the key names are in fact identical `test_name: tcp_connect` and `tcp_connect: []`.

I do agree it's confusing though :P

Reply:
The tricky part is in the key names indeed. Let's just keep the comments in the code as reminders.